packages/frontend/core/src/modules/docs-search/worker/out-worker.ts

```diff
@@ -0,0 +1,98 @@
+import { DebugLogger } from '@affine/debug';
```

Comment:
what are the meaning of `in/out-worker`?

Reply:
in-worker: run in the worker
out-worker: run outside of the worker